tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -306,25 +306,6 @@ public void EnsureAllRabbitMQManifestTypesHaveVersion0Suffix()
         Assert.Equal("container.v0", server.GetProperty("type").GetString());
     }
 
-    [Fact]
-    public void EnsureAllKafkaManifestTypesHaveVersion0Suffix()
```

Comment:
I don't think this test adds value over the `AddKafkaTests.VerifyManifest` test. That test verifies it is a container.v0 resource.

Reply:
ok :)